django_summernote/templates/django_summernote/widget_common.html

```diff
@@ -143,7 +143,7 @@
 {% endif %}
     });
 }
-{% if not lazy %}
+{% if not config.lazy %}
```

Comment:
With `lazy` set to `True`, I believe this line breaks iframe widgets

Reply:
Before this change, loading the iframe didn't have access to to the `lazy` param, so it properly always called the `initSummernote_<id>` function. Now that it always has access to it, `lazy` set to `True` breaks this.

Reply:
I'm guessing we want something including the `iframe` value, like:
```
{% if not iframe and not config.lazy %}
```
```